test/offsetarray.jl

```diff
@@ -313,7 +313,7 @@ seek(io, 0)
 @test readdlm(io, eltype(A)) == parent(A)
 
 amin, amax = extrema(parent(A))
-@test clamp(A, (amax+amin)/2, amax) == clamp(parent(A), (amax+amin)/2, amax)
+@test clamp.(A, (amax+amin)/2, amax).parent == clamp.(parent(A), (amax+amin)/2, amax)
```

Comment:
Oh, interesting, guess this is a bugfix then. Should there be a few more tests of this behavior?